Driver/Driver/Helper/Animator.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Animator.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 22.02.2023.
+//
+
+import UIKit
+
+public class Animator {
+    public static let sharedInstance = Animator()
+    var transpranView = UIImageView()
```

Comment:
should be transparent. Please make it correct.

Reply:
Do we need to define it as UIImageView, UIView is enough for this purpose IMO

Reply:
UIView should be enough.